frontend/src/employee-mobile-frontend/common/TopBar.tsx

```diff
@@ -23,8 +23,11 @@ import { LoggedInUser } from './top-bar/LoggedInUser'
 import { TopBarIconContainer } from './top-bar/TopBarIconContainer'
 import { currentSystemNotificationQuery } from './top-bar/queries'
 
-const StickyTopBar = styled.section<{ invertedColors?: boolean }>`
-  position: sticky;
+const StickyableTopBar = styled.section<{
+  invertedColors?: boolean
+  sticky?: boolean
+}>`
```

Comment:
Kun tyylitellään HTML-elementtejä, pitäisi ei-standardien propertyjen nimien eteen laittaa `$`. Siis tässä tapauksessa `$invertedColors` ja `$sticky`. Tällöin styled-components osaa jättää ne passaamatta läpi DOM:iin.

Meillä on käytössä [proppien automaattinen filtteröinti](https://github.com/espoon-voltti/evaka/blob/master/frontend/src/citizen-frontend/App.tsx#L72-L81), mutta sei ei ole 100% tarkka ja sen käyttöä [ei enää suositella](https://styled-components.com/docs/faqs#shouldforwardprop-is-no-longer-provided-by-default).